channeldb/models/channel_edge_policy.go

```diff
@@ -60,6 +60,9 @@ type ChannelEdgePolicy struct {
 	// HTLCs for each millionth of a satoshi forwarded.
 	FeeProportionalMillionths lnwire.MilliSatoshi
 
+	// InboundFee that will be charged for incoming HTLCs.
+	InboundFee
```

Comment:
I had that thought too. An alternative I had in mind was to define an InboundFee in UpdatePolicy and pass a pointer to it to UpdateEdge. This would be quickly adjusted. I would just need to know in which direction we want to go.

Reply:
Have created an alternative. Preview can be found here: https://github.com/feelancer21/lnd/compare/e080792451184cf0810831230cf8ef8609a4cc51..9b17806

Reply:
I'd probably have it this way (I think you had a similar version before): https://github.com/lightningnetwork/lnd/commit/b8b2a05605af4dc9587392a56e9dd9e1182347b4, but not sure about the other reviewer's opinions. We could deal with adding the inbound fees field in a later PR. I think technically `htlc_maximum_msat` was also part of the extra data, because it was optional, but got then promoted to `models.UpdateEdgePolicy` and `lnwire.ChannelUpdate`. We could do a similar transition by removing inbound fees from the extra data to promote it to `models.UpdateEdgePolicy`. We could then leave any non-interpreted extra data untouched, which is useful to display in `describegraph`.

Reply:
Ok. Have integrated the version of @bitromortac.

Reply:
One problem I wanted to note here (not an issue with this PR) with extracting the inbound fees (or other data later) from the extra data may be that the order of TLVs could change if we re-serialize to `lnwire.ChannelUpdate`, maybe leading to signature invalidation, when we try to relay the message.